q2_alignment/plugin_setup.py

```diff
@@ -19,7 +19,10 @@
     citation_text=("MAFFT multiple sequence alignment software version 7: "
                    "improvements in performance and usability. Katoh K, "
                    "Standley DM. Mol Biol Evol. 2013 Apr;30(4):772-80. "
-                   "doi: 10.1093/molbev/mst010.")
+                   "doi: 10.1093/molbev/mst010."),
+    description=('This QIIME 2 plugin provides support for generating '
+                 'and working with sequence alignments.'),
+    short_description='Plugin for generating and working with alignments.'
```

Comment:
Should we use `manipulating` here (like masking)? Since it's really `q2-phylogeny` that "works with" the alignment.